scripts/screeps_tencent_batch_rl_runner.py

```
@@ -1064,6 +1115,94 @@ def bash_lc(script: str) -> str:
+            "status": "shadow",
+            "liveEffect": False,
+            "officialMmoWrites": False,
+            "officialMmoWritesAllowed": False,
+            "billingGuardBeforeScale": True,
+            "sshControllerOnlyExpected": args.controller_ip,
+            "secretsPrinted": False,
+        },
+    }
+
+
+def validate_scale_proof_result(raw: Any, expected_environments: int) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `raw`

(ANN401)